server/migrations/1620932907414_add-otps.js

```diff
@@ -0,0 +1,31 @@
+/* eslint-disable camelcase */
+const { dbClient, collections, schema } = require('../db');
+
+exports.up = async () => {
+  await schema.relationalTables.map((item) => dbClient.runRawQuery(item.definition));
+
+  await dbClient.db.query(
+    `CREATE INDEX IF NOT EXISTS email_address_index ON ${collections.CONFIRM_INTEREST}(email_address);`
+  );
+
+  await dbClient.db.query(
+    `CREATE INDEX IF NOT EXISTS otp_index ON ${collections.CONFIRM_INTEREST}(otp);`
+  );
+
+  await dbClient.db
+    .query(`INSERT INTO ${collections.CONFIRM_INTEREST} (email_address)
+  SELECT DISTINCT p.body ->> 'emailAddress' AS email
+      FROM participants p 
+      WHERE 
+          p.body ->> 'emailAddress' NOT IN (
```

Comment:
Seems like we could do this without the `NOT IN` subquery thing. Suggestion incoming.

Reply:
Can we try something like this?
```
SELECT DISTINCT participants.body->>'emailAddress' AS email
    FROM participants
    JOIN participants_status 
    ON participants.id = participants_status.participant_id
    AND participants_status.status != 'hired'
    AND (
        (participants.body->>'interested')::TEXT = 'yes'  
        OR (participants.body->>'interested')::TEXT IS NULL
    );
```